examples/js/loaders/GLTF2Loader.js

```diff
@@ -2692,7 +2694,11 @@ THREE.GLTF2Loader = ( function () {
 										break;
 
 									default:
-										child = new THREE.Mesh( originalGeometry, material );
+									    // save morph targets before replace
+                                        // meshes
+                                        var morph = child.morphTargetInfluences;
+                                        child = new THREE.Mesh( originalGeometry, material );
+                                        child.morphTargetInfluences = morph;
```

Comment:
^These five lines are not needed now, a fix upstream has fixed that part of the issue. (you may not see the fix without rebasing, but that's fine)